### PR TITLE
Prepend fabric name

### LINKED ISSUE #184: Fix run_map for diff_run framework
The diff_run framework currently creates a run_map file under `collections/ansible_collections/cisco/nac_dc_vxlan/roles/validate/files/run_map.yml` which will work for a single fabric but this file needs to prepend the name of the fabric to avoid namespace collisions.

### PR DESCRIPTION
**Pull request by mtarking**
Prepend fabric name to prevent over-writing incorrect file and fabric map.

Resolves #184

### CODE REVIEW

File: plugins/action/common/read_run_map.py [hunk 1]
```diff
@@ -39,13 +39,15 @@ def run(self, tmp=None, task_vars=None):
         results = super(ActionModule, self).run(tmp, task_vars)
         results['diff_run'] = True
 
+        fabric_name = task_vars['inventory_hostname']
```
Comment (devegupt) [inline plugins/action/common/read_run_map.py:42]: I executed a few tests with updating the fabric name under vxlan.global.name, I couldn't see the new run_map file with fabric name prefix. As per this line in the code, I understand we are actually prepending inventory name and not the fabric name with run_map.yml file.

**Review by devegupt — COMMENTED** [review]
Added a comment in read_run_map.py.

File: plugins/action/common/read_run_map.py (hunk 1 above)
Comment (mtarking) [reply plugins/action/common/read_run_map.py:42]: @devegupt interesting point you raise; lets discuss in the standup.

**Review by juburnet — CHANGES_REQUESTED** [review]

File: plugins/action/common/read_run_map.py (hunk 1 above)
Comment (mtarking) [reply plugins/action/common/read_run_map.py:42]: Right, that's expected with that commit. Before modifying the plugin, the test was to make sure things indeed still worked correctly. I'll modify the code now.

**Pull request merged**